### PR TITLE
Feature/upgrade v2.0

### PR DESCRIPTION
**Pull request by rklaver**

### CODE REVIEW

File: TestListener.php [hunk 1]
```diff
@@ -1,157 +0,0 @@
-<?php
-
-use Doctrine\Bundle\DoctrineBundle\Command\CreateDatabaseDoctrineCommand;
```
Comment (Slurp) [inline TestListener.php:3]: Why remove this? This should be a part of a default project?

File: app/phpunit.xml.dist [hunk 2]
```diff
@@ -22,22 +32,6 @@
         <log type="junit" target="../build/logs/junit.xml" logIncompleteSkipped="false"/>
     </logging>
 
-    <php>
```
Comment (Slurp) [inline app/phpunit.xml.dist:25]: Why is this removed?

File: composer.json [hunk 3]
```diff
@@ -12,16 +12,20 @@
...
-        "doctrine/doctrine-migrations-bundle": "~1.1.1",
-        "friendsofsymfony/user-bundle": "2.0.*@dev",
-        "kriswallsmith/spork": "~0.3.0",
-        "bureaublauwgeel/kunstmaanadminbundle": "3.6.0.7",
-        "bureaublauwgeel/routable-entity-bundle": "dev-develop",
-        "bureaublauwgeel/generatorbundle": "dev-develop",
-        "bureaublauwgeel/websitesettingsbundle": "dev-develop"
+        "friendsofsymfony/http-cache-bundle": "~1.3",
+        "doctrine/doctrine-migrations-bundle": "~1.3",
+        "friendsofsymfony/user-bundle": "2.0.0-beta1",
```
Comment (Slurp) [inline composer.json:23]: Does this work with the current Kunstmaan version?

**Review by Slurp — CHANGES_REQUESTED** [review]

File: composer.json (hunk 3 above)
Comment (rklaver) [reply composer.json:23]: Yes, in fact this is the exact version required in Kunstmaan 3.6.0.1. See: https://github.com/Kunstmaan/KunstmaanBundlesCMS/blob/3.6.1/composer.json#L31

File: TestListener.php (hunk 1 above)
Comment (rklaver) [reply TestListener.php:3]: This should only be added to a project if we do integration tests, since that is when you need a local database to be created. We should try to encourage writing unit tests instead.

File: app/phpunit.xml.dist (hunk 2 above)
Comment (rklaver) [reply app/phpunit.xml.dist:25]: Most of it was actually just moved up in the file.

**Review by Slurp — APPROVED** [review]
After discussion we will merge this.
